### DISCUSSION TITLE
Paste submit type button clashing with bootstrap styles/behaviour

### DISCUSSION THREAD

**Discussion by mirjammalksoo** [twilio-labs/paste#1355] [Q&A] [answered]
Hi!
 
Our team discovered a problem/behaviour with Paste `submit` type buttons when page/menu layout includes bootstrap in the assets (for example `console` layout).

When clicking on the submit button and using `event.preventDefault()`, it goes into custom bootstrap loading state and stays like this. 

![image](https://user-images.githubusercontent.com/40227184/114399149-47b8be00-9ba9-11eb-8db8-1d6661d1fbd8.png)

However we would instead like to use the Paste loading state during request processing and then enable the button again after the request has finished:

![image](https://user-images.githubusercontent.com/40227184/114399249-5f904200-9ba9-11eb-8711-acaf594c4908.png)

I am wondering if you have seen this issue before or might know how it could be solved?

Thanks!

**Comment by vankhoawin** [comment]
i've noticed this before, it's more of a Console issue than a Paste issue. have you tried using `event.stopPropagation()`?

**Comment by SiTaggart** [comment, marked as answer]
Hi @mirjammalksoo  

Seems to come from Console-js. Can you try adding `data-no-spinner="true"` to the button and seeing is that disables the behavior?

**Comment by mirjammalksoo** [comment]
Hi @vnguyen94 and @SiTaggart!

I tried out both solutions and they both disabled the behaviour. Thanks for helping me out with this!
